pippo-core/src/main/java/ro/pippo/core/Application.java

```diff
@@ -376,6 +425,17 @@ public static Application get() {
         }
     }
 
+    private Route GET(StaticResourceHandler resourceHandler) {
+        if (getRouter().uriPatternFor(resourceHandler.getClass()) != null) {
+            throw new PippoRuntimeException("You may only register one route for {}",
```

Comment:
We should eliminate this check.  There are valid cases where you may want to register two different handlers of the same type.